cmd/metrics-server/app/start.go

```diff
@@ -25,14 +25,15 @@ import (
 
 	cliflag "k8s.io/component-base/cli/flag"
 	"k8s.io/component-base/logs"
+	_ "k8s.io/component-base/logs/json/register"
 	"k8s.io/component-base/term"
 	"sigs.k8s.io/metrics-server/cmd/metrics-server/app/options"
 )
 
 // NewMetricsServerCommand provides a CLI handler for the metrics server entrypoint
 func NewMetricsServerCommand(stopCh <-chan struct{}) *cobra.Command {
 	opts := options.NewOptions()
-
+	o := logs.NewOptions()
```

Comment:
Please include options as part of Metrics Server options.

Reply:
ok，I will modify it, thanks .

Reply:
done